fiftyone/server/query.py

```diff
@@ -190,7 +190,7 @@ def modifier(doc: dict) -> dict:
         doc["frame_fields"] = _flatten_fields([], doc.get("frame_fields", []))
         doc["brain_methods"] = list(doc.get("brain_methods", {}).values())
         doc["evaluations"] = list(doc.get("evaluations", {}).values())
-        doc["saved_views"] = doc.get("saved_views", [])
+        doc["saved_views"] = doc.get("saved_views", doc.get("views", []))
```

Comment:
I think you will need to update the `schema.graphql` and add the new field to the dataset queries. There are two, right now. Probably should only be one.
* https://github.com/voxel51/fiftyone/blob/develop/app/packages/core/src/Dataset.ts
* https://github.com/voxel51/fiftyone/blob/develop/app/packages/relay/src/mutations/setView.ts

Reply:
good catch. line 193 should be reverted back to `doc["saved_views"] = doc.get("saved_views", [])`. 

we have views in db initially - then changed to saved_views. I was testing around